HOWTO-make-a-release.md

```diff
@@ -282,44 +303,49 @@ The do-release.pl script will display the commands you will need to issue to
 send the announcement emails later.  Keep a note of those commands for
 future reference.
 
-Verify that the tarballs are available via FTP:
+For public releases, verify that the tarballs are available via FTP:
 
-    ftp://ftp.openssl.org/source/
+    ls /srv/ftp/source
 
-And that they are ready for the website:
+For premium releases, verify that the tarballs are available via SFTP:
 
-    ls /var/www/openssl/source
+    ls /srv/premium
 
-*For OpenSSL 3.0 and on*, push your local changes to the main source repo as
-instructed by `dev/release.sh`.  You may want to sanity check the pushes by
-inserting the `-n` (dry-run) option.
+*For OpenSSL 3.0 and on*, push your local changes to the appropriate source
+repo as instructed by `dev/release.sh`.  You may want to sanity check the
+pushes by inserting the `-n` (dry-run) option.
 
-*For OpenSSL before 3.0*, simply push your local changes to the main source
-repo, and please do remember to push the release tags as well. You may want to
-sanity check the pushes by inserting the `-n` (dry-run) option. You must specify
-the repository / remote and tag to be pushed:
+*For OpenSSL before 3.0*, simply push your local changes to the appropriate
+source repo, and please do remember to push the release tags as well. You
+may want to sanity check the pushes by inserting the `-n` (dry-run)
+option. You must specify the repository / remote and tag to be pushed:
 
     git push <repository> <tagname>
 
 ## Updating the release data
 
-Push the newsflash changes to the release data repo.  When you do this, the
-website will get updated and a script to flush the Akamai CDN cache will be
-run.  You can look at things on www-origin.openssl.org; the CDN-hosted
-www.openssl.org should only be a few minutes delayed.
+If there is a PR against the release repo to be merged, perform the merge
```

Comment:
Previously this referred to the "release data repo"...the word "data" has been dropped which kind of seems important. When I first read this I was temporarily confused.....what "release repo" are we talking about here?

Reply:
Fixed